backend/local-test-services/Tiltfile

```diff
@@ -6,6 +6,9 @@ k8s_yaml(kustomize('../auth/auth-api/k8/overlays/minikube'))
 k8s_resource('auth-api', port_forwards=8080)
 docker_build('eu.gcr.io/insight/auth-api', '..', {}, '../auth/auth-api/docker/Dockerfile.jvm')
 
+k8s_yaml('../auth/auth-api/migrations/manifests.yaml')
+docker_build('eu.gcr.io/insight/auth-api-migrations', '../auth/auth-api/migrations', {})
```

Comment:
The way tilt works, this will be automatically run on every code change in migrations folder. But this might not be what we want (image trying to work on the migrations code). Each change will rebuild, rerun and try to apply "same" migration version with different code...